tools/data_usage/console.cpp

```diff
@@ -1,109 +1,139 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
-#include <windows.h>
 #include <iostream>
 #include <string>
+#include <windows.h>
+#include "bpf.h"
 #include "ebpf_api.h"
+#include "libbpf.h"
 #include "ebpf_nethooks.h"
 
-#define MAP_COUNT 1
+typedef struct _app_name
+{
+    wchar_t name[32];
+} app_name_t;
 
 int main(int argc, const char** argv)
 {
     UNREFERENCED_PARAMETER(argc);
```

Comment:
Better to comment out the variable names than use UNREFERENCED_PARAMETER
https://www.osgwiki.com/wiki/Cpp_Core_Guidelines#I.MSFT.22:_Delete.2C_comment_out_or_use_maybe_unused_on_unreferenced_parameters